php/yii/controllers/SiteController.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace app\controllers;
+
+use yii\web\Controller;
+
+class SiteController extends Controller
+{
+    public $layout = false;
+    public $enableCsrfValidation = false;
```

Comment:
I see no need to disable it. It should not affect anything significantly.

Reply:
It brokes POST request:
POST /user, SHOULD return a successful status code and an empty string